servicemanager/smprocess.py

```diff
@@ -165,7 +165,6 @@ def _get_process_ids_for_processes_matching(regex):
     ps_command = subprocess.Popen(command, shell=True, stdout=subprocess.PIPE, universal_newlines=True)
     stdout, _ = ps_command.communicate()
     rv = list(map(int, stdout.split("\n")[:-1]))
-    stdout.close()
```

Comment:
What effect did this line have previously? Closing things is generally a good thing...

Reply:
stdout here turned out to be a string of the content of stdout so closing it just threw a method not found error.